### PR TITLE
Create labeler.yml

### PR DESCRIPTION
**Pull request by alexearnshaw**
## Describe the changes

- This PR adds a workflow for the Github action https://github.com/CodelyTV/pr-size-labeler and labels pull requests with a size label according to the lines of code changed.
- This PR uses a bot user and custom token instead of the standard Github token for actions so should also work for pull requests from forks. (as suggested in https://github.com/CodelyTV/pr-size-labeler/issues/8)
- For info I used an existing `docsbuilder` service account that was already a member of Axway org and the steps at the bottom of [this link](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token) to create the token and secret for that account.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [x] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [x] You have applied a production label if this needs a manual push to Zoomin
* [x] You have added it to the correct GitHub project
* [x] You have added it to the correct Sprint milestone
* [x] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [x] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [x] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Comment by alexearnshaw** [comment]
@andreamussap This seems to be working as you can see in this screenshot:
image
However, I can't test it for PRs from forks until it's merged.

**Comment by andreamussap** [comment]
> @andreamussap This seems to be working as you can see in this screenshot:
> However, I can't test it for PRs from forks until it's merged.

PR Approved. About the test, did you create it from a forked branch? I hope it works!!

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    name: Label the PR size
+    steps:
+      - uses: codelytv/pr-size-labeler@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.DOCSBUILDER_TOKEN }}
+          xs_max_size: '10'
+          s_max_size: '100'
```
Comment (andreamussap) [inline .github/workflows/labeler.yml:14]: @alexearnshaw if this labe is **s_max_size** how come it label your PR with **size/S**?

Comment (alexearnshaw) [reply .github/workflows/labeler.yml:14]: @andreamussap turns out these are not the labels, these are just variables I think? The labels are defined somewhere else in the action  and are the standard ones afaik

**Pull request merged**